src/assets/scss/MasonryLayout.scss

```diff
@@ -12,3 +12,16 @@
   box-sizing: border-box;
   min-height: 100vh;
 }
+
+.masonry-layout {
+  column-count: 2;
+  column-gap: 10px;
```

Comment:
I think this should be converted to `rem`, for the sake of consistency

Reply:
Done